### ISSUE TITLE
Put the h2 first in .features > article > header

### ISSUE THREAD

**Issue by icaaq** [h5bp/html5please#90]
To make each article more semantic the h2 should be moved to the top in the header-element. This will help e.g people that uses screen readers to understand what each article is about.
